conduct/code_of_conduct.md

```diff
@@ -73,10 +73,9 @@ You can file a report by emailing
 information on reporting in-person at an event, please see our Reporting
 Guidelines.
 
-You have the option to keep your report anonymous or request that we
-follow up with you directly. Note that we cannot follow up on an
-anonymous report, but we will take appropriate action to prevent a
-recurrence.
+The online form gives you the option to keep your report anonymous or request
+that we follow up with you directly. While we cannot follow up on an anonymous
+report, we will take appropriate action to prevent a recurrence.
```

Comment:
I would end the sentence at `...action.` Otherwise, this implies that the report was a violation.